lib/database/session-manager.ts

```
@@ -0,0 +1,247 @@
+
+  async getSessionLogs(sessionId: string, limit: number = 50): Promise<any[]> {
+    return db
+      .select()
+      .from(sessionLogs)
+      .where(eq(sessionLogs.sessionId, sessionId))
+      .orderBy(desc(sessionLogs.timestamp))
+      .limit(limit);
+  }
+
+  // Cleanup operations
+  async cleanupOldSessions(olderThanDays: number = 30): Promise<number> {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)